packages/cdk8s-cli/src/cli/cmds/init.ts

```diff
@@ -44,11 +44,11 @@ async function determineDeps(version: string, dist?: string): Promise<Deps> {
     const ret = {
       npm_cdk8s: path.resolve(dist, 'js', `cdk8s@${version}.jsii.tgz`),
       npm_cdk8s_cli: path.resolve(dist, 'js', `cdk8s-cli-v${version}.tgz`), // yarn pack adds a "v" before the version
-      npm_cdk8s_plus: path.resolve(dist, 'js', `cdk8s-plus@${version}.jsii.tgz`),
+      npm_cdk8s_plus: path.resolve(dist, 'js', `cdk8s-plus-17@${version}.jsii.tgz`),
```

Comment:
Curious what we'll do with this

Reply:
Maybe just a dynamic search for `cdk8s-plus*`

Reply:
Let's figure out later